README.md

```diff
@@ -1,100 +1,137 @@
-# OSTIS-web-platform
-
+# OSTIS Web Platform
 <img src="https://github.com/ostis-ai/ostis-web-platform/actions/workflows/main.yml/badge.svg?branch=develop">
```

Comment:
Can we make own badge? (maybe, OSTIS or SC-code, or LaTeX, or another one)

Reply:
We can also add license badge

Reply:
[shields.io](https://shields.io/) is a great website which has tons of badges. You can check out the possible options here, though I don't think we need a badge of our own for now. A license badge is a good idea though, I'll add it